advanced/subpath/route53-cloudfront.mdx

```diff
@@ -208,4 +231,4 @@ Now, we're going to bring the functionality of the CloudFront distribution into
 
 Your documentation is now live at `/docs` for your primary domain.
 
-<Propagating />
+<Propagating />
```

Comment:
```suggestion
```

Can this be deleted? Not sure what it is and I think it's an artifact from the writer bot.